gridstatus/ercot.py

```diff
@@ -303,6 +303,18 @@ def get_energy_storage_resources(self, date="latest", verbose=False):
 
         df = pd.DataFrame(data["previousDay"]["data"] + data["currentDay"]["data"])
 
+        # todo(kanter): fix this for future DST dates
+        if "2024-11-03 02:00:00-0600" in df["timestamp"].values:
+            # ERCOT publishes two intervals with 2am timestamp
+            # during CDT to CST transition
+            # but skips the repeated 1am timestamp
+            # let's manually fix this before further timestamp parsing
+            df.loc[
+                (df["timestamp"] == "2024-11-03 02:00:00-0600")
+                & (df["dstFlag"] == "N"),
```

Comment:
This assumes the `dstFlag == "Y"` for the repeated hour only?

Reply:
not 100% I follow the question.  but there two 2am intervals with -6 offset but a missing 1am interval with -6 offset, so i am manually shifting one based on `dstFlag`

Reply:
How do you know to use "Y" or "N" to identify the interval to shift?

Reply:
If `dstFlag == "Y"` for the repeated hour and ERCOT labels rows by hour end then this logic makes sense.

Reply:
yeah, it is dstFlag == "Y" when things repeat. 

this data is 5 minute though. every other interval gets labeled correctly. this is some weird error on ERCOT's side  i think